lib/widgets/molecules/upcoming_session_tile.dart

```diff
@@ -1,46 +1,45 @@
 import 'package:flutter/material.dart';
-import 'package:flutter_gen/gen_l10n/app_localizations.dart';
 import 'package:intl/intl.dart';
 import 'package:mm_flutter_app/constants/app_constants.dart';
-import 'package:mm_flutter_app/widgets/atoms/rectangle_button.dart';
 
 class UpcomingSessionTile extends StatelessWidget {
   static const Duration joinMinimumTime = Duration(minutes: 5);
+  final String avatarUrl;
   final DateTime dateTime;
   final String mentorName;
 
   const UpcomingSessionTile({
     Key? key,
+    required this.avatarUrl,
     required this.dateTime,
     required this.mentorName,
   }) : super(key: key);
 
-  List<Widget> _createSessionButtons(AppLocalizations l10n) {
-    List<Widget> buttons = [];
-    // Add JOIN button only if session is coming up soon.
-    if (DateTime.now().isAfter(dateTime.subtract(joinMinimumTime))) {
-      buttons.add(Padding(
-        padding: const EdgeInsets.only(
-          right: Insets.widgetSmallInset,
-        ),
-        child: RectangleButton(
-          onPressed: () => {},
-          text: l10n.homeUpcomingSessionJoin,
-        ),
-      ));
-    }
-    buttons.add(RectangleButton(
-      onPressed: () => {},
-      text: l10n.homeUpcomingSessionReschedule,
-    ));
-    return buttons;
-  }
+  // List<Widget> _createSessionButtons(AppLocalizations l10n) {
+  //   List<Widget> buttons = [];
+  //   // Add JOIN button only if session is coming up soon.
+  //   if (DateTime.now().isAfter(dateTime.subtract(joinMinimumTime))) {
+  //     buttons.add(Padding(
+  //       padding: const EdgeInsets.only(
+  //         right: Insets.widgetSmallInset,
+  //       ),
+  //       child: RectangleButton(
+  //         onPressed: () => {},
+  //         text: l10n.homeUpcomingSessionJoin,
+  //       ),
+  //     ));
+  //   }
+  //   buttons.add(RectangleButton(
+  //     onPressed: () => {},
+  //     text: l10n.homeUpcomingSessionReschedule,
+  //   ));
+  //   return buttons;
+  // }
```

Comment:
Feel free to delete stuff that is no longer needed. There is nothing groundbreaking here and we can always go back to previous versions if we need to.

Reply:
Done!